contracts/MemeFactory.sol

```diff
@@ -7,7 +7,7 @@ import {IUniswapV3} from "./interfaces/IUniswapV3.sol";
 
 interface IBuyBackBurner {
     function checkPoolPrices(address nativeToken, address memeToken, address uniV3PositionManager, uint24 fee,
-        uint256 allowedDeviation, bool isNativeFirst) external view;
+        bool isNativeFirst) external;
```

Comment:
This function is now not view, and the allowed deviation must be set there, not here.